examples/operations/QueueScript.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>add an s to check</p><p> </p><p>Consider:</p><p>The next two cells execute some liveliness and state checks to ensure the script can run successfully.</p><p> </p><p>I assume that's why you're doing it? If not, then just say why you need to do them. Only so someone might understand why you do it here but not for all scripts etc.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>